packages/table/src/Row/InternalRowBase.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import { cx } from '@leafygreen-ui/emotion';
+import { useDarkMode } from '@leafygreen-ui/leafygreen-provider';
+
+import { clickableStyles, disabledStyles, rowBaseStyles } from './Row.styles';
+import { InternalRowBaseProps } from './Row.types';
+
+/**
+ * The base Row component, extended by `InternalRow(With/Without)RT`
+ */
+const InternalRowBase = ({
```

Comment:
should we handle darkmode here, since both with and without RT rely on it?

Reply:
The theme var is consumed throughout RowWithRT and RowWithoutRT, so it wouldn't make sense to apply all styles here. For example, you can't use CSS `:nth-of-type(even)` with virtual rows, since the zebra style is dependent on the index of the virtual row, not the index of the tr in the DOM.